Library/Homebrew/extend/os/linux/requirements/osxfuse_requirement.rb

```diff
@@ -0,0 +1,37 @@
+require "requirement"
+
+class OsxfuseRequirement < Requirement
+  download "https://github.com/libfuse/libfuse"
```

Comment:
Yeh, let's pass then. Perhaps a potential Homebrew/brew PR is in your future to handle that 😀